### DISCUSSION TITLE
Regarding the Traefik Dashboard option

### DISCUSSION THREAD

**Discussion by jln-dk** [Dokploy/dokploy#895] [General]
What does the "Enable Dashboard" option in the `Settings > Server > Traefik` dropdown do? I don't see it mentioned in the docs anywhere.

**Comment by jln-dk** [comment]
Nevermind, found it here: https://github.com/Dokploy/dokploy/discussions/696

Would be good to add to the docs.

**Reply by nktnet1** [reply]
Thanks for linking, was helpful.

I've also replied with steps for how to run it on a custom domain (figured it out through Coolify docs)
- https://github.com/Dokploy/dokploy/discussions/696#discussioncomment-11658370
